sorc/ncep_post.fd/CALPW.f

```diff
@@ -289,12 +289,14 @@ SUBROUTINE CALPW(PW,IDECID)
         DO J=JSTA,JEND
           DO I=1,IM
              DP      = PINT(I,J,L+1) - PINT(I,J,L)
-             PW(I,J) = PW(I,J) + Qdum(I,J)*DP*GI*HTM(I,J,L)
-            IF (IDECID == 17 .or. IDECID == 20 .or. IDECID == 21) THEN
-             PW(I,J) = PW(I,J) + Qdum(I,J)*MAX(DP,0.)*GI*HTM(I,J,L)
-            ENDIF
+
             IF (IDECID == 19) THEN
              PW(I,J) = PW(I,J) + Qdum(I,J)
+            ELSE
+           !  PW(I,J) = PW(I,J) + Qdum(I,J)*DP*GI*HTM(I,J,L)
+           ! IF (IDECID == 17 .or. IDECID == 20 .or. IDECID == 21) THEN
```

Comment:
If the lines are not needed, you should delete these lines.

Reply:
deleted unnecessary lines